randovania/data/json_data/prime2.txt

```diff
@@ -1187,13 +1246,9 @@ Asset id: 3032508327
                   Morph Ball
                   Any of the following:
                       Power Bomb
-                      Morph Ball Bomb and Difficulty: Trivial
+                      Morph Ball Bomb and Bomb Jump (Trivial) and Difficulty: Trivial
   > Dynamo Chamber/Event - Dynamo Chamber Gates
-      All of the following:
-          Scan Visor and Before Dynamo Chamber Gates and Difficulty: Trivial
-          Any of the following:
```

Comment:
The items are not needed?

Reply:
If you notice a few lines down, I changed that all to only needing the new tub opening the gate from behind, because you can scan the terminal from behind.